filebeat/reader/readjson/docker_json.go

```diff
@@ -136,7 +137,9 @@ func (p *DockerJSONReader) Next() (reader.Message, error) {
 					return message, err
 				}
 				message.Content = append(message.Content, next.Content...)
+				partialBytes += next.Bytes
```

Comment:
I believe you don't need the aux variable here, you could just do `message.Bytes += next.Bytes`, what do you think?

Reply:
You are right :) I'll fix it. Thank you!